lib/galaxy/webapps/base/static

```diff
@@ -0,0 +1 @@
+../../../../static
```

Comment:
Is that meant to be here ?

Reply:
(and if yes, do we need to backport this ?)

Reply:
Good eye -- unrelated (possible) bugfix for something Kaivan and I were looking at that I didn't notice got lumped in here.  Harmless, but it shouldn't be in this PR.